jenkins/scripts/slurm_run.sh

```
@@ -27,54 +37,35 @@
 else
     while [ ! -f install_lock.lock ]; do
         sleep 5
     done
 fi
-testList="$testList_$splitId"
-export CPP_TEST_TIMEOUT_OVERRIDDEN=$pytestTestTimeout
-export LLM_ROOT=$llmSrcNode
-export LLM_MODELS_ROOT=$MODEL_CACHE_DIR
+
 export UCX_TLS=^gdr_copy
 cd $llmSrcNode/tests/integration/defs
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)